--- a/vendor/laragear/webauthn/src/Models/WebAuthnCredential.php
+++ b/vendor/laragear/webauthn/src/Models/WebAuthnCredential.php
@@ -95,7 +95,7 @@ class WebAuthnCredential extends Model
     protected $casts = [
         'counter' => 'int',
         'transports' => 'array',
-        'public_key' => 'encrypted',
+        'public_key' => 'string',
         'certificates' => 'array',
         'disabled_at' => 'timestamp',
     ];
